src/main/java/com/uber/cadence/internal/tracing/TracingPropagator.java

```diff
@@ -64,6 +64,8 @@ public Span spanForExecuteWorkflow(DecisionContext context) {
 
     return tracer
         .buildSpan(EXECUTE_WORKFLOW)
+        .ignoreActiveSpan() // ignore active span to start a new trace that ONLY links the start
```

Comment:
Hmm, are we sure this does what we want it to do? Ignoring the current span will prevent two coroutines running on the same thread from inadvertently picking up each others' context state, but then how/where do we track the context state if not on the threadlocal?

Reply:
It's linked explicitly by `asChild`

Reply:
It's linked explicitly by `asChild`